app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,76 @@
+class TweetsController < ApplicationController
+
+  get '/tweets/new' do # displays 'create tweet' form
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do # created tweet is submitted to database
+    if params[:content] == "" # checks submission for content
```

Comment:
Thanks! I appreciate the suggestion. I'll definitely look to implement this in future instances.